app-text/msort/msort-8.53.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+inherit autotools
+
+DESCRIPTION="A program for sorting files in sophisticated ways"
+HOMEPAGE="https://billposer.org/Software/msort.html"
+SRC_URI="https://billposer.org/Software/Downloads/${P}.tar.bz2"
+
+KEYWORDS="~amd64 ~x86"
+LICENSE="GPL-3"
+SLOT="0"
+IUSE="heap +icu +libuninum"
+
+RDEPEND="dev-libs/tre
+	icu? ( dev-libs/icu:= )
+	!icu? ( dev-libs/libutf8proc:= )
+	libuninum? ( dev-libs/libuninum:= )"
+
+DEPEND="${RDEPEND}"
+
+PATCHES=(
+	"${FILESDIR}/${P}-find-utf8proc-headers.patch"
+	"${FILESDIR}/${P}-fix-icu-build.patch"
+	"${FILESDIR}/${P}-fix-missing-limits-header.patch"
+	"${FILESDIR}/${P}-fix-segfault-on-certain-input.patch"
+	"${FILESDIR}/${P}-respect-users-cflags.patch"
+	)
+
+src_prepare() {
+	default
+
+	eautoreconf
+}
+
+src_configure() {
+	local myeconfargs=(
+		$(use_enable !heap allocaok)
```

Comment:
Are you sure alloca is a good default? I'm not really sure what they're doing but I hope this isn't going to overflow the stack on big input, is it? ;-)

Reply:
To be honest, I don't have a clou. The build systems uses allocaok as default and has an optional optional for enabling heap instead of allocaok. Should I set instead heap as default?